bots/employee-finder/src/SSW.SophieBot/Properties/launchSettings.json

```diff
@@ -18,7 +18,7 @@
     "BotProject": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "http://localhost:3978",
+      "applicationUrl": "http://localhost:3980",
```

Comment:
This local setting change should not be checked in

Reply:
I changed this because the composer lunch at port 3980. So they can be consistent.